src/main/java/org/reaktivity/nukleus/sse/internal/stream/ServerStreamFactory.java

```diff
@@ -161,6 +178,7 @@ public ServerStreamFactory(
         this.sseTypeId = supplyTypeId.applyAsInt(SseNukleus.NAME);
         this.setHttpResponseHeaders = this::setHttpResponseHeaders;
         this.setHttpResponseHeadersWithTimestampExt = this::setHttpResponseHeadersWithTimestampExt;
+        this.challengeEventType = config.getEventType();
```

Comment:
After changing the config name and return type to `String`, this becomes `new StringFW(config.getChallengeEventType())`.